languages/sr_RS/Zend_Captcha.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * sr_RS-Revision: 28.October.2015
+ */
+return array(
+    // Zend\Captcha\ReCaptcha
+    "Missing captcha fields" => "Недостају captcha поља",
+    "Failed to validate captcha" => "Неуспела captcha валидација",
```

Comment:
Looks like we imported this word as well https://sr.wikipedia.org/wiki/%D0%9A%D0%B0%D0%BF%D1%87%D0%B0.